---
layout: archive
title: ""
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

# Hanfeng Lin
*Baylor College of Medicine, Houston, Texas 77030, US*
*hanfeng.lin@bcm.edu *

---

## EDUCATION
* **Ph.D. in Chemical, Physical and Structural Biology** | Jan 2021 – est. 2025
    * *Baylor College of Medicine, Houston, Texas, United States*
* **BSc in Biochemistry and Pharmacology, First Class Honour Degree** | Sept 2019 - Jun 2020
    * *University of Strathclyde, Glasgow, UK*
* **B.S. in Pharmacy (Biochemistry and Pharmacology)** | Sept 2016 - Jun 2020
    * *China Pharmaceutical University, Nanjing, China*
    * GPA: 3.88 / 4.0

---

## RESEARCH EXPERIENCE

### Graduate Research Assistant
*Baylor College of Medicine, Houston, TX | Mentored by Prof. Jin Wang | Jun 2021 - Present*

* **Research Focus 1: Develop a Chemoproteomics Platform to Profile Covalent Ligand Binding Kinetics: Covalent Occupancy Kinetics Enrichment via Proteomics (COOKIE-Pro)** (January 2023 – Present)
    * Developed the COOKIE-Pro method for proteome-wide kinetic profiling, integrating chemoproteomics and mass spectrometry to assess covalent inhibitor binding kinetics ($k_{inact}$ and $K_I$).
    * Identified high-affinity off-target kinases such as TEC, BTK, and BLK, advancing covalent fragment screening.
    * Optimized covalent inhibitor design using kinetic modeling, introducing a two-step fitting process to estimate kinetic parameters with improved accuracy and reduced sample size. Ranked fragment-cysteine interactions based on ligand efficiency (LE) and ligand lipophilic efficiency (LLE).
    * Pioneered a protein-level enrichment workflow using desthiobiotin probes to improve the specificity and depth of covalent inhibitor profiling, contributing new insights into warhead chemistry and target engagement.

* **Research Focus 2: Discovery and Mechanistic Characterization of Novel VAV1 Molecular Glue Degraders** (June 2024 – Present)
    * Identified VAV1 as a novel neosubstrate for CRBN molecular glues using an unbiased, high-throughput global proteomics screening approach.
    * Discovered and validated a non-canonical RT-loop degron within the VAV1 SH32 domain (RDxS motif), expanding the known mechanisms for CRBN-mediated degradation beyond typical G-loop motifs.
    * Demonstrated that conformational restriction of the degrader molecule via strategic halogenation markedly potentiates degradation efficacy by pre-organizing the molecule for ternary complex formation.
    * Revealed that a single molecular glue can target distinct degron types by identifying LIMD1 as an off-target that utilizes a canonical G-loop, underscoring the necessity of proteomic profiling.

* **Research Focus 3: Expanding Cereblon Neosubstrates Beyond G-Loops: A Novel Molecular Glue Strategy for Kinase Degradation** (January 2023 – Present)
    * Discovery of a unique and potent BTK degrader with quick degradation kinetics. Further biochemical assay confirmed its molecular glue property.
    * Confirmed BTK is a CRBN Neosubstrate Without G-Loop Degrons using both MD simulation methods and Cryo-EM, revealing broad and novel PPI residue network.
    * Mutational experiment and structure-activity relationship (SAR) confirmed BTK glue ternary complex formation is contributed from both ligand and PPI.
    * Developed a novel chemoproteomics strategy to discover lead-glues to degrade other neosubstrate kinases.

* **Research Focus 4: Develop a Novel Chemoproteomics-Based Covalent Fragment Drug Discovery Platform** (November 2022 – Present)
    * Developed an innovative strategy to enhance the detection of cysteine residues by conjugating desthiobiotin directly on covalent fragments, improving sensitivity in chemoproteomics for low-abundance proteins.
    * Identified a novel RCC2 covalent binder and its reactive cysteine target within a cryptic pocket, validated with biochemical mode-of-action studies.
    * Constructed a covalent fragment and probe library (~100 compounds) with drug-like properties, targeting carcinogenesis pathways for future medicinal chemistry efforts.

* **Research Focus 5: Rational Design of Bifunctional Molecules for Targeted Protein Degradation** (June 2021 – June 2023)
    * Developed a PROTAC-induced ternary complex *in silico* modeling pipeline using a linker-centric approach and parallel molecular dynamics (MD) simulation.
    * Investigated the impact of linker design in PROTAC molecules on degradation potency, offering insights through in-silico modeling and establishing a structure-activity relationship (SAR) for protein degradation drugs.
    * My integrated computational pipeline successfully predicted and provided clues for a cryo-EM structure solution of an in-house PROTAC.

### Undergraduate Thesis Project
*University of Strathclyde, Glasgow, UK | Instructed by Prof. Nigel J Pyne | Oct 2019 – Feb 2020*
*Cellular Effects Caused by Cancer-related S1P2 and S1P4 Receptor Agonists in HEK293 cell*
* The project aimed to explore whether protease cleavage can activate the S1P receptor via a special mechanism by removing the N terminal domain, which in turn activate downstream signaling pathway.

### Group leader
*China Pharmaceutical University (CPU), Nanjing, China | Instructed by Dr. Yucheng Zhao | Mar 2019 – Sept 2019*
*Identification of Benzyl Isoquinoline Alkaloids (BIA) Genes in Corydalis by Full-length Transcriptome Data Analysis*
* Obtained the first full-length transcriptome of Corydalis, analyzed the related transcripts of enzymes and verified that all BIA-synthetic proteins in Corydalis root had their corresponding mRNA.
* Classified O-methyltransferase (OMT) protein family by constructing phylogenetic tree.
* A first-author research paper published.

### Core member
*China Pharmaceutical University (CPU), Nanjing, China | Instructed by Prof.Yibei Xiao | Mar 2019 – Aug 2019*
*Biophysical Study on Proteins Related to CRISPR System and Their Activities*
* Constructed the expression strains and purified key proteins Cas9, Cas1, Csn2 and Cas13a in CRISPR system.
* Obtained RNA by in-vitro transcription of spacer cDNA sequences using purified T7 transcriptase, and researched their binding and activity.
* Analyzed the structure of the complex (PLK1-PBD + peptide drug) with biophysical methods (X-ray crystallography).

### Project leader
*China Pharmaceutical University (CPU), Nanjing, China | Instructed by Dr. Jiyang Xu | May 2018 – Apr 2019*
*Effect of tripeptidase gene knockout by Red recombinant system on glutathione biosynthesis*
* Constructed engineering bacteria *E. coli* pET28a-gsh I and pBAD24-gsh II, screened positive recombinant strains and knock out the tripeptidase gene by using Red recombination system.
* Inhibited the expression of tripeptidase and GSH degradation, determined the content of GSH by alloxan method.
* Compared data of the glutathione production from the knockout strain and constructed strain, verified knocking out the tripeptidase gene could increase the GSH production.

---

## PUBLICATION

### Research Articles
1.  **Lin H**, Yang B, Ding L, Yang YY, Holt MV, Jung SY, Zhang B, Wang MC, Wang J. COOKIE-Pro: Covalent Inhibitor Binding Kinetics Profiling on the Proteome Scale. *Nature Communications*, **2025**, Manuscript.
2.  **Lin, H.**, Yu, X., Cheng, R., Qi, X., Yang, Y.-Y., Zhou, S., Che, X. & Wang, J. Beyond the G-Loop: CRBN Molecular Glues Potently Target VAV1 via a Novel SH3 RT-Loop Degron. *Nature Chemical Biology*, **2025**, Manuscript.
3.  Cheng R†, **Lin H**†, et al. Expanding Cereblon Neosubstrates Beyond G-Loops: A Novel Molecular Glue Strategy for Kinase Degradation. **2025**, Manuscript. †: *These authors contribute equally to this work*.
4.  Cheng R†, **Lin H**†, et al. Covalent PROTACs achieve sub-nanomolar potency and selective BTK degradation independent of catalytic turnover. **2025**, Manuscript.
5.  **Lin H**, Riching K, Lai MP, Lu D, Cheng R, Qi X, Wang J. Lysineless HiBiT and NanoLuc Tagging Systems as Alternative Tools for Monitoring Targeted Protein Degradation. *ACS Med Chem Lett.* **2024** Jul 28;15(8):1367-1375.
6.  **Lin H**, Wang L, Jiang X, Wang J. Glutathione Dynamics in Subcellular Compartments and Implications for Drug Development. *Curr Opin Chem Biol.* **2024** Aug;81:102505.
7.  Xu D†, **Lin H**†, Zhang K, Li C, Zhao X, Xu J, Zhao Y. Integration of Full-Length Transcriptomics and Targeted Metabolomics to Identify Benzylisoquinoline Alkaloid Biosynthetic Genes in *Corydalis yanhusuo*. *Hortic Res.* **2021** Jan 10;8(1):16. †: *These authors contribute equally to this work*.
8.  Zheng, H., **Lin, H.**, Alade, A.A., Chen, J., Monroy, E.Y., Zhang, M. & Wang, J. AlphaFold3 in drug discovery: A comprehensive assessment of capabilities, limitations, and applications. *bioRxiv*, **2025**.
9.  Wang L, **Lin H**, Yang B, Jiang X, Chen J, Roy Chowdhury S, Cheng N, Nakata PA, Lonard DM, Wang MC, Wang J. Development of a Novel Amplifiable System to Quantify Hydrogen Peroxide in Living Cells. *J Am Chem Soc.* **2024** Aug 14;146(32):22396-22404.
10. Sun Z, **Lin H**, Hu L, Neetu N, Sankaran B, Wang J, Prasad BVV, Palzkill T. Klebsiella pneumoniae Carbapenemase Variant 44 Acquires Ceftazidime-Avibactam Resistance by Altering the Conformation of Active-Site Loops. *J Biol Chem.* **2024** Jan;300(1):105493.
11. Yu X, **Lin H**, Li F, Wang J, Lu D. Development of Biochemical and Cellular Probes to Study RIPK1 Target Engagement. *ACS Med Chem Lett.* **2024** May 10;15(6):906-916.
12. Yu X, Lu D, Qi X, **Lin H**, Holloman BL, Jin F, Xu L, Ding L, Peng W, Wang MC, Chen X, Wang J. Development of a First-in-Class RIPK1 Degrader to Enhance Antitumor Immunity. *Nat Commun.* **2024**.
13. Yu X, Guo W-H, **Lin H**, Cheng R, Monroy EY, Jin F, Ding L, Lu D, Qi X, Wang MC, Wang J. Discovery of a Potent BTK and IKZF1/3 Triple Degrader Through Reversible Covalent BTK PROTAC Development. *Curr Res Chem Biol.* **2022**; 2:100029.
14. Lu D, Yu X, **Lin H**, Cheng R, Monroy EY, Qi X, Wang MC, Wang J. Applications of Covalent Chemistry in Targeted Protein Degradation. *Chem Soc Rev.* **2022** Nov 14;51(22):9243-9261.
15. Lu S, Hu L, **Lin H**, Judge A, Rivera P, Palaniappan M, Sankaran B, Wang J, Prasad BVV, Palzkill T. An Active Site Loop Toggles Between Conformations to Control Antibiotic Hydrolysis and Inhibition Potency for CTX-M β-Lactamase Drug-Resistance Enzymes. *Nat Commun.* **2022** Nov 7;13(1):6726.
16. Bao Z, Zhang K, **Lin H**, Li C, Zhao X, Wu J, Nian S. Identification and Selection of Reference Genes for Quantitative Transcript Analysis in *Corydalis yanhusuo*. *Genes*. **2020**;11(2):130.
17. Wang Z, Xu J, Liu Y, Chen J, **Lin H**, Huang Y, Bian X, Zhao Y. Selection and Validation of Appropriate Reference Genes for Real-Time Quantitative PCR Analysis in *Momordica charantia*. *Phytochemistry*. **2019** Apr;164:1-11.
18. Zhang J, Gao D, Li Q, Zhao Y, **Lin H**, Bi Q, Zhao Y. Biodegradation of Polyethylene Microplastic Particles by the Fungus *Aspergillus Flavus* From the Guts of Wax Moth *Galleria Mellonella*. *Sci Total Environ.* **2020** Feb;704:135931.
19. Wang L, Yu X, Xu W, Zhang J, **Lin H**, Zhao Y. Complete Chloroplast Genome Sequencing Supports *Angelica decursiva* as an Independent Species From *Peucedanum praeruptorum*. *Physiol Mol Biol Plants.* **2021** Nov;27(11):2503-2515.

### Conference Posters
* **Lin H**, Yang B, Cheng R, Lu D, Jung SY, Wang J. COOKIE-Pro: A Chemoproteomics Method for Covalent Inhibitor Binding Kinetics Profiling. Drug Discovery Chemistry 2024. San Diego, CA. (*Nominated for Poster Finalist*).
* Yang, K., **Lin, H.**, Larsen, B., Sun, H., Hakimi, A., Pekar Hart, T. & Wang, J. High-throughput compound screening workflow for targeted protein degradation on an Orbitrap Astral mass spectrometer with accurate label-free quantitation. American Society of Mass Spectrometry 2025, Baltimore, MD.
* Cheng R, **Lin H**, Lu D, Wang J. Development of a Highly Potent Covalent BTK PROTAC: Sub-Stoichiometric Degradation is Dispensable. Drug Discovery Chemistry 2024. San Diego, CA.
* **Lin H**, Lu D, Cheng R, Yu X, Jin F, Ding L, Wang J. E3 Ligase Complex Structure-Based Rational Design Accelerating the Lead Optimization of PROTACs. Promega Target Protein Degradation Conference 2023. Madison, WI.

### Oral Presentation
* Developing A Novel Chemoproteomics Based Covalent Drug Discovery Platform. *SCBA-TX Annual Symposium, 2024. Houston, TX.*

---

## HONORS & AWARDS
* Texas Advanced Computing Center Leadership Resource Allocation (LRAC)
* Baylor College of Medicine – Full tuition scholarship
* University of Strathclyde First Class Honours in BSc Biochemistry and Pharmacology
* China Pharmaceutical University The First-class Scholarship of Exploring World Plan
* 2nd China Undergraduate Life Science Competition (CULSC) Third-prize
* National Scholarship
* Model of Triple-A student -- the highest individual honor for students at CPU
* The Honorary Title of Triple-A student
* First-class Scholarship
* The Honorary Title of Outstanding Volunteer

---

## SKILLS

* **Lab Skills**: DIA/DDA/PRM proteomics (TMT & label-free), IP-MS, NanoBRET, TR-FRET, Western Blot, seamless plasmid cloning, cell culture, luciferase-based assay, lentivirus, PCR, qPCR, CRISPR knockout, Protein purification, FPLC (Gel filtration & Ion exchanger), PAGE (SDS, Native, Urea), X-ray crystallography, flow cytometry, LC-MS, mouse experiment.
* **Computational**:
    * **Software**: GROMACS for MD simulation, Schrodinger Maestro & Desmond, Gaussian, OpenEye Omega, Rosetta Suite, Proteome Discoverer, DIA-NN, Skyline, Snapgene, PyMOL, Coot, Phenix, MATLAB SimBiology, BioEdit, BLAST, MegaX, UCSF Chimera.
    * **Scripting**: Python, R, Bash.
    * **Statistics and Figure**: GraphPad Prism, R, limma, matplotlib, UMAP, Origin, SPSS, Adobe Photoshop, Adobe Illustrator, ChemDraw.
* **Languages**:
    * **Chinese**: Native
    * **English**: Fluent
    * **Japanese**: Moderate
    
<script>
    window.onload = function () {
        setTimeout(function () {
            var archive = document.querySelector("#main>.archive");
            var children = archive.children;
            var kill = children[children.length - 1];
            archive.removeChild(kill);
        }, 1000);
    }
</script>

<script src="https://v1.cnzz.com/z_stat.php?id=1278007712&web_id=1278007712"></script>

